ansible/doc/README.testbed.Overview.md

```diff
@@ -54,3 +54,35 @@ DUT front panel port is directly connected to one of PTF container ports. Usuall
 ![](img/testbed-injected.png)
 
 DUT front panel port is directly connected to one of VMs interfaces. But also we have a tap into this connection. Packets coming from the physical vlan interface are sent to both the VMs and the PTF docker. Packets from the VM and PTF docker are sent to the vlan interface. It allows us to inject packets from the PTF host to DUT and maintain a BGP session between VM and DUT at the same time.
+
+# SONiC Testbed with Keysight IxNetwork as Traffic Generator
+
+This section gives an overview of the stand-alone testbed topology where SONiC DUT is directly connected with Keysight’s protocol emulator and traffic generator (IxNetwork).
+
+## Physical Topology
+
+Based on test need there may be multiple topologies possible as shown below :
+
+- Single DUT Topology
+![](img/single-dut-topology.png)
+
+
+- Multiple DUT Topology
+![](img/multiple-dut-topology.png)
+
+
+- Multiple IxNetwork Topology
```

Comment:
Could you please explain the difference between `Multiple DUT Topology` and `Multiple IxNetwork Topology`? I cannot tell any difference based on the images.

Reply:
I’ve updated Multiple IxNetwork Topology ( https://github.com/SubhajitPalKeysight/sonic-mgmt/blob/master/ansible/doc/img/multiple-ixnetwork.PNG ).

Multiple DUT Topology : Let’s  consider an Ixia chassis has max N test ports. If a test topology has a SUT (multiple DUTs) and total port requirement is <=N, then single Ixia Chassis (IxNetwork) is sufficient to test such topology containing multiple DUTs. There is no need to have multiple Ixia Chassis.

Multiple IxNetwork Topology : In a test topology which requires > N ports, multiple Ixia Chassis will be needed. Such a system can be managed by a single IxNetwork API Server capable of spawning multiple sessions.

Reply:
would you please review and let us know your comment.